src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -0,0 +1,148 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.exception.UnhandleableHearingState;
+import uk.gov.hmcts.reform.sscs.model.HearingState;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class HearingsServiceTest {
+
+    final String hearingRequestId = "12345";
+    final String hmcStatus = "TestStatus";
+    final Number version = 123;
+    final String cancellationReasonCode = "TestCancelCode";
+
+    final HearingsService hearingsService = new HearingsService();
+
+    @DisplayName("processHearingRequest Parameterized Tests")
+    @ParameterizedTest
+    @CsvSource(value = {
+        "READY_TO_LIST,CRATE_HEARING",
+        "READY_TO_LIST,UPDATE_HEARING",
+        "READY_TO_LIST,UPDATED_CASE",
+        "READY_TO_LIST,CANCEL_HEARING",
+        "READY_TO_LIST,PARTY_NOTIFIED",
+        "READY_TO_LIST,null",
+        "null,CRATE_HEARING",
+        "CREATE_DRAFT,UPDATE_HEARING",
+        "null,null",
+    }, nullValues = {"null"})
+    void processHearingRequest(EventType event, HearingState state) throws UnhandleableHearingState {
+        // TODO Finish Test when method done
+        SscsCaseData caseData = SscsCaseData.builder()
+                .ccdCaseId("12345")
+                .appeal(Appeal.builder().build())
+                .build();
+        HearingWrapper wrapper = HearingWrapper.builder()
+                .event(event)
+                .state(state)
+                .originalCaseData(caseData)
+                .updatedCaseData(caseData)
+                .build();
+
+        if (nonNull(event)) {
```

Comment:
I think this test is trying to be too many things at once. You should split out the null/empty cases, and the different switch paths.